server/handlers/policies_handlers.go

```diff
@@ -0,0 +1,106 @@
+package handlers
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/fs"
+	"net/http"
+	"os"
+	"path"
+	"path/filepath"
+
+	"github.com/gorilla/mux"
+	"github.com/layer5io/meshkit/models/meshmodel"
+	"github.com/layer5io/meshkit/models/meshmodel/core/types"
+	"github.com/layer5io/meshkit/models/meshmodel/core/v1alpha1"
+)
+
+func (h *Handler) GetAllMeshmodelPolicy(rw http.ResponseWriter, r *http.Request) {
+	rw.Header().Add("Content-Type", "application/json")
+	enc := json.NewEncoder(rw)
+	typ := mux.Vars(r)["model"]
+	name := mux.Vars(r)["name"]
```

Comment:
where are you getting the name from if this handler is commented? https://github.com/meshery/meshery/pull/7064/files#diff-c9f0fd2985523c81296d5a3b0b745ef02fa9d539a883a799977e0f4cb73f45e1R165

Reply:
fixed.